templates/andrzejki.html

```diff
@@ -0,0 +1,61 @@
+<html>
+    <head>
+        <title>Andrzejki</title>
+        <style>
+            body { background-color: black; }
+            .images {
+                display: grid;
+                grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+                justify-items: center;
+                align-items: center;
+                height: 100vh;
+                width: 100%;
+                max-height: 100%;
+                padding: 0;
+            }
+            img {
+                width: calc(100% - 1em);
+                height: auto;
+                object-fit: contain;
+                image-rendering: pixelated;
+            }
+        </style>
+        <script>
+            const IMAGES_COUNT = 5;
+            const URL = "/image.png";
+            const TIME = 5000;
```

Comment:
what time is it? time for dinner :D? I assume this time set number of milliseconds to load next image to get wow effect. What do you think about changing it to DELAY_TO_LOAD_IMAGE?